exec/java-exec/src/main/java/org/apache/drill/exec/ops/BufferManager.java

```diff
@@ -51,11 +48,12 @@ public BufferManager(BufferAllocator allocator, FragmentContext fragmentContext)
     this.fragmentContext = fragmentContext;
   }
 
+  @Override
   public void close() throws Exception {
-    Object[] mbuffers = ((LongObjectOpenHashMap<Object>)(Object)managedBuffers).values;
-    for (int i =0; i < mbuffers.length; i++) {
+    final Object[] mbuffers = ((LongObjectOpenHashMap<Object>) (Object) managedBuffers).values;
+    for (int i = 0; i < mbuffers.length; i++) {
       if (managedBuffers.allocated[i]) {
-        ((DrillBuf)mbuffers[i]).release();
+        ((DrillBuf) mbuffers[i]).release(1);
```

Comment:
It saves a function call; seems desirable in tight loops and low-level value vector situations.